core/tests/test_utils.py

```diff
@@ -386,6 +386,21 @@ class TestBase(unittest.TestCase):
         'next_node_id': 'node_2'
     }
 
+    VERSION_2_STORY_CONTENTS_DICT = {
```

Comment:
Isn't this used only in story_services_test? If  so, add it to that file instead of here.

Reply:
I just followed #9095 to implement it here. So should I change it?